### PR TITLE
Add Vagrantfile to run Fluentd in Linux box very easily

### PR DESCRIPTION
**Pull request by tagomoris**
especially for OSX users

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+rake_target = ENV['TEST'] ? "test TEST=#{ENV['TEST']}" : "test"
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.network "private_network", type: "dhcp"
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo apt-get install software-properties-common
```
Comment (cosmo0920) [inline Vagrantfile]: Should we use  `apt-get install -y` in this line?
This line works for `ubuntu/trusty64`, though.

File: Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    sudo apt-add-repository ppa:brightbox/ruby-ng
+    sudo apt-get update
+    sudo apt-get install -y git ruby2.3 ruby2.3-dev
+    sudo apt-get install ruby-switch
+    sudo ruby-switch --set ruby2.3
+    gem install bundler
+    (cd /vagrant; bundle install; bundle exec rake #{rake_target})
```
Comment (cosmo0920) [inline Vagrantfile]: It seems that this installation process does not install gcc.
Is it enough for testing?

File: Vagrantfile (hunk 1 above)
Comment (tagomoris) [inline Vagrantfile]: Right.

File: Vagrantfile (hunk 2 above)
Comment (tagomoris) [inline Vagrantfile]: It sounds good to add these.

Comment (tagomoris) [inline Vagrantfile]: Adding `build-essential` only looks to work.

**Pull request merged**